### PR TITLE
mergeback release 7.0.0

### PR DESCRIPTION
**Pull request by lionce**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,27 @@
...
+
+- \(MODULES-10362\) handle authenticationinfo consistently [\#253](https://github.com/puppetlabs/puppetlabs-iis/pull/253) ([pillarsdotnet](https://github.com/pillarsdotnet))
+- \(MODULES-10361\) sort https bindings first [\#252](https://github.com/puppetlabs/puppetlabs-iis/pull/252) ([pillarsdotnet](https://github.com/pillarsdotnet))
+
+## [v6.0.0](https://github.com/puppetlabs/puppetlabs-iis/tree/v6.0.0) (2019-10-30)
+
+[Full Changelog](https://github.com/puppetlabs/puppetlabs-iis/compare/v5.0.0...v6.0.0)
+
 ## v5.0.0
 All notable changes to this project will be documented in this file.
```
Comment (alexjfisher) [inline CHANGELOG.md:27]: This should be removed from the `HISTORY.md` file.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,27 @@
...
+- \(FM-8195\) Convert to Litmus [\#249](https://github.com/puppetlabs/puppetlabs-iis/pull/249) ([florindragos](https://github.com/florindragos))
+
+### Fixed
+
+- \(MODULES-10362\) handle authenticationinfo consistently [\#253](https://github.com/puppetlabs/puppetlabs-iis/pull/253) ([pillarsdotnet](https://github.com/pillarsdotnet))
+- \(MODULES-10361\) sort https bindings first [\#252](https://github.com/puppetlabs/puppetlabs-iis/pull/252) ([pillarsdotnet](https://github.com/pillarsdotnet))
+
+## [v6.0.0](https://github.com/puppetlabs/puppetlabs-iis/tree/v6.0.0) (2019-10-30)
+
+[Full Changelog](https://github.com/puppetlabs/puppetlabs-iis/compare/v5.0.0...v6.0.0)
```
Comment (op-ct) [inline CHANGELOG.md:24]: A git log diff [isn't a useful changelog](https://keepachangelog.com/en/1.0.0/#log-diffs), and it [doesn't meet the criteria](https://github.com/puppetlabs/puppetlabs-iis/pull/260/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eL3-L4) documented at the top of this file:

> "All notable changes to this project will be documented in this file. The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org)."

1. No notable changes are documented for 6.0.0.
2. The 6.0.0 entry doesn't follow the "[Keep a Changelog](http://keepachangelog.com/en/1.0.0/)" format.
   * There's no indication what was added, fixed, or changed between 5.0.0 and 6.0.0
3. The v6.0.0 release doesn't appear to have followed [Semantic Versioning](http://semver.org/):
   * There's no indication of what API-breaking changes prompted the major version bump from 5.0.0 to 6.0.0
   * As far as I could tell after digging through the "Full Changelog" link, it appears that nothing did:
     * There were no functional code changes (just documentation under `lib/` and test updates under `spec/`).
     * The module's API didn't change.
     * The module's dependencies didn't change.

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,27 @@
...
+
 ## v5.0.0
 All notable changes to this project will be documented in this file.
```
Comment (op-ct) [inline CHANGELOG.md:28]: * The "All notable changes..." blurb is redundant with the same notice at the top of the file.
 * The date format for entries changed from ` - YYYY-MM-DD` (the [KACL recommendation](https://keepachangelog.com/en/1.0.0/#confusing-dates)) before v5.0.0 to ` (YYYY-MM-DD)` after v5.0.0.
 * The v5.0.0 entry doesn't have a date at all.

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,27 @@
 
 All notable changes to this project will be documented in this file. The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org).
 
+## [v7.0.0](https://github.com/puppetlabs/puppetlabs-iis/tree/v7.0.0) (2020-01-21)
```
Comment (op-ct) [inline CHANGELOG.md:5]: This project uses [Semantic Versioning](http://semver.org)—what was the API-breaking change that bumped the major version from 6.0.0 to 7.0.0?

**Review by op-ct — CHANGES_REQUESTED** [review]

**Review by DavidS — APPROVED** [review]
mergeback approval

**Pull request merged**